src/components/atoms/OakGrid/OakGrid.tsx

```diff
@@ -18,6 +22,7 @@ const gridStyle = css<OakGridProps>`
     (props) => props.$gridTemplateColumns,
   )}
     ${responsiveStyle("grid-template-rows", (props) => props.$gridTemplateRows)}
+    ${responsiveStyle("grid-gap", (props) => props.$gridGap, parseSpacing)}
```

Comment:
`grid-gap` [is a deprecated name](https://developer.mozilla.org/en-US/docs/Web/CSS/gap#:~:text=Early%20versions%20of%20the%20specification%20called%20this%20property%20grid%2Dgap).

We already use the correct names for `row-gap` and `column-gap` so we should probably use the unprefixed `gap` property here.